actual-rest-api/openapi.yaml

```diff
@@ -0,0 +1,142 @@
+openapi: 3.0.0
+info:
+  title: University.Schedule.api
+  version: '1.0'
+  license:
+    name: The Unlicense
+    url: 'https://raw.githubusercontent.com/modelix/modelix-samples/main/LICENSE'
+servers:
+  - url: 'http://localhost:8080'
+    description: localDev
+paths:
+  /rooms:
+    get:
+      summary: Provides access to all known rooms
+      tags: []
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/RoomList'
+      operationId: get-rooms
+  /lectures:
+    get:
+      summary: Provides access to all known lectures
+      tags: []
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/LectureList'
+      operationId: get-lectures
+  '/rooms/{roomRef}':
+    parameters:
+      - $ref: '#/components/parameters/roomRef'
+    get:
+      summary: Provides access to a single room
+      tags: []
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Room'
+      operationId: get-rooms-roomID
+  '/lectures/{lectureRef}':
+    parameters:
+      - $ref: '#/components/parameters/lectureRef'
+    get:
```

Comment:
```suggestion
    get:
      operationId: getLecture
```

Reply:
Ah, these operationIds are auto generated and already included below. I'll propose another round of more readable changes.